app/domain/models/base/BaseGetOneModel.scala

```diff
@@ -0,0 +1,6 @@
+package domain.models.base
+
+trait BaseGetOneModel {
```

Comment:
@Bukarachka as mentioned before, let's use "findBy" naming here. Also, not sure if we need "Base" as a prefix for models